records_mover/DRIVERS.md

```diff
@@ -0,0 +1,130 @@
+    your `./itest` runs.
+3. Fix these "singledb" tests!
+  * Now that you have tests running (and failing), you can address the
+    problems one by one.  Here are things you are likely to need to
+    do--I'd suggest waiting for the problem to come up via the test
+    and then applying the fix until the tests pass.  If you encounter
+    things not on the list below, add them here for the next person!
+  * Add Python driver (either SQLAlchemy or if SQLAlchemy supports it
+    natively, maybe just the DBAPI driver) as a transtive dependency
+    in `setup.py`.  Rerun `./deps.sh` and then `./itest --docker
+    build` to re-install locally.
+  * If database connections aren't working, you may want to insert
+    some debugging into `records_mover/db/connect.py` to figure out
+    what's going on.
+  * Access errors trying to drop a table in the `public` schema:
+    Probably means whatever default schema comes with your database
+    user doesn't match the default assumption - modify
+    `tests/integration/records/single_db/base_records_test.py` to
+    match.
+  * `NotImplementedError: Please teach me how to integration test
+    mysql`: Add information for your new database in
+    `tests/integration/records/records_database_fixture.py`.
+  * `AssertionError: ['INTEGER(11)', 'VARCHAR(3)', 'VARCHAR(3)',
+    'VARCHAR(1)', 'VARCHAR(1)', 'VARCHAR(3)', 'VARCHAR(111)', 'DATE',
+    'TIME', 'DATETIME', 'DATETIME']`: Double check the types assigned.
+    You may need to subclass DBDriver and implement to convince
+    records mover to create the types you expect.
+  * Errors from `tests/integration/records/directory_validator.py`:
+     ```console
+     AssertionError:
+     received ['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'time', 'datetime', 'datetime'],
+     expected [['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'time', 'datetime', 'datetimetz'], ['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'string', 'datetime', 'datetimetz']]
+     ```
+
+     To address, make sure the types returned are as expected for this database.
+   * `KeyError: 'mysql'`:
+     `tests/integration/records/single_db/test_records_numeric.py`
+     needs to be modified to set expectations for this database type.
+     You can set this to 'bluelabs' as we haven't yet taught
+     records-mover to do bulk imports, so we have no idea what the
+     ideal records format variant is for that yet.
+
+4. If there are things you see below that you know are needed from the
+   above list, but the tests are passing, consider adding an
+   integration test to match.
+5. Edit
+   `tests/integration/records/multi_db/test_records_table2table.py` to
+   include the new test database and run `./itest table2table` to run
+   tests.  Fix errors as they pop up:
+   * TBD
+7. Add support for bulk import if the database supports it TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker